package.json

```diff
@@ -39,7 +39,7 @@
     "klaw": "^3.0.0",
     "lodash.clonedeep": "^4.5.0",
     "mime-types": "^2.1.24",
-    "parcel-bundler": "1.12.3"
+    "parcel": "^2.0.0-beta.2"
```

Comment:
is there an eta on release ? just wondering if we want to rely on a beta

Reply:
Unfortunately no ETA. Just that it has been in beta for about a year now.